R/dataDrivenClusters.R

```diff
@@ -5,34 +5,44 @@
 dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
```

Comment:
After the roxygen documentation, we should specify somewhere that the input data must be in a wide format as specified in my email. I think this will need to go in the description of the function where you describe what the function does. Most brain data does not automatically look this way, so the user needs to be told very clearly how the data should be structured.